tests/main/snapshot-basic/task.yaml

```diff
@@ -17,6 +17,8 @@ execute: |
     SET_ID=$( snap save test-snapd-tools test-snapd-rsync | cut -d\  -f1 | tail -n1 )
 
     # check it includes both snaps
+    snap saved | grep test-snapd-tools
```

Comment:
Why not `MATCHES`?

Reply:
Right.. Because I've just copied and tweaked the existing checks of this test. Updated, but I'm not touching existing greps in this test to keep the diff at minimum.